package.json

```diff
@@ -67,6 +67,7 @@
   "dependencies": {
     "firebase": "^9.17.2",
     "libphonenumber-js": "^1.10.24",
+    "livekit-client": "^1.9.3",
```

Comment:
I think it would be a good idea to include it as an external dependency, like React is included.